src/onegov/election_day/formats/common.py

```diff
@@ -97,6 +99,7 @@ def load_csv(
             encoding=encoding,
             rename_duplicate_column_names=rename_duplicate_column_names
         )
+        # FIXME: Remove if not necessary to speed up
```

Comment:
Why would this be needed? This just consumes the generator (and it would be consumed again the next time you loop over the lines). Are you looping over all records to see if there's an error? If not, then you should remove this.

Reply:
The solution was to call `next(csv.lines)` to raise the `EmptyLineInFileError`